bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/AbstractRetryingRpcListener.java

```diff
@@ -174,7 +175,16 @@ public void onClose(Status status, Metadata trailers) {
         status.getCause(), failedCount, status, channelId);
 
     rpc.getRpcMetrics().markRetry();
-    retryExecutorService.schedule(this, nextBackOff, TimeUnit.MILLISECONDS);
+    retryExecutorService.schedule(getRunnable(), nextBackOff, TimeUnit.MILLISECONDS);
+  }
+
+  protected Runnable getRunnable() {
+    return new Runnable() {
+      @Override
+      public void run() {
+        AbstractRetryingRpcListener.this.run();
+      }
+    };
```

Comment:
If we are supporting java 8 only, then this can be simplified to 
```java
retryExecutorService.schedule(this::run, nextBackOff, TimeUnit.MILLISECONDS);
```